src/styles/Document.ts

```diff
@@ -1,6 +1,6 @@
 import { cssRule } from 'typestyle';
 
-cssRule('html, body', {
+cssRule('html, body, .perf-body', {
   margin: 0,
   padding: 0,
 });
```

Comment:
@Carla-Moz you suggested to add here the background. I noticed this should change in dark mode. Should we resolve this here or in another PR?
`cssRule('html, body, .perf-body', {
  margin: 0,
  padding: 0,
  background: #ffffff
});
`

Reply:
Yes, you're right! Please remove that change: `'.perf-body' `and the` background: #ffffff.` I forgot I solved this in a previous PR so once you merge, it will be updated.